vowpalwabbit/vw_versions.h

```diff
@@ -22,4 +22,5 @@
 #define VERSION_FILE_WITH_CB_TO_CBADF "8.11.0"  // first version that maps --cb to use cb_adf
 #define VERSION_FILE_WITH_REG_CB_SAVE_RESUME "8.11.0"     // version that ensures accuracy of loaded models in regcb
 #define VERSION_FILE_WITH_SQUARE_CB_SAVE_RESUME "8.11.0"  // version that ensures accuracy of loaded models in squarecb
-#define VERSION_FILE_WITH_FIRST_SAVE_RESUME "8.11.0"      // version that ensures accuracy of loaded models in first
\ No newline at end of file
+#define VERSION_FILE_WITH_FIRST_SAVE_RESUME "8.11.0"      // version that ensures accuracy of loaded models in first
+#define VERSION_FILE_WITH_RANDOM_SEED "8.11.1"            // version which saves random seed from vw in model file
```

Comment:
we are about to release 8.11.0 so I think you can just re-use that version here, no need to use 8.11.1 as far as I can tell

Reply:
I used 8.11.1 to ensure that model files before this could still be read properly, without the random seed. @rajan-chari do you have thoughts on this?

Reply:
but we haven't released 8.11.0 yet we can't jump to 8.11.1